documentation/modules/ref-operator-topic.adoc

```diff
@@ -0,0 +1,107 @@
+// Module included in the following assemblies:
+//
+// assembly-using-the-topic-operator.adoc
+
+[id='ref-operator-topic-{context}']
+= Kafka topic resource
+
+The `KafkaTopic` resource is used to configure topics, including the number of partitions and replicas.
+
+The full schema for `KafkaTopic` is described in xref:type-KafkaTopic-reference[`KafkaTopic` schema reference].
+
+= Identifying a Kafka cluster for topic handling
```

Comment:
Should this be `==`?

Reply:
I was looking at it from a TOC point of view. Maybe we can drop some of these sections to a lower level. I'll have another look.